src/pybind/mgr/dashboard_v2/tox.ini

```diff
@@ -37,7 +37,7 @@ commands =
     bash ../src/vstart.sh -n
     sleep 10
     python ./bin/ceph config-key set mgr/dashboard_v2/x/server_port 9865
-    python ./bin/ceph mgr module enable dashboard_v2
+    python ./bin/ceph mgr module enable dashboard_v2 --force
```

Comment:
I guess you added this here to be able to load the module even without having the `python-coverage` package install. I don't think this is a good idea because this will also prevent us from catching errors from other reasons. It's very important that we can test if the module is correctly enabled.

Reply:
No. This was a non-deterministic error.

Reply:
Nonetheless, we should not pass the `--force` flag, it's preferable to have spurious false failures than not catching real failures